forloop_modules/queries/db_model_templates.py

```diff
@@ -121,6 +121,23 @@ class DeleteUidObject(BaseModel):
 #     project_uid: str = ""
 #     last_active_pipeline_uid: Optional[str] = None
 
+class JobSortableColumnsEnum(str, Enum):
+    STATUS = "status"
+    CREATED_AT = "created_at"
+    STARTED_AT = "started_at"
+    COMPLETED_AT = "completed_at"
+    PIPELINE_UID = "pipeline_uid"
```

Comment:
Pipeline_uid sounds inconsistent with the other attributes, are you sure about that?